includes/class-comment-walker.php

```diff
@@ -170,4 +169,115 @@ protected function avatar_only( $comment, $depth, $args ) {
+
+	/**
+	 * Outputs a comment in the HTML5 format.
+	 *
+	 * @param WP_Comment $comment Comment to display.
+	 * @param int        $depth   Depth of the current comment.
+	 * @param array      $args    An array of arguments.
+	 */
+	protected function html5_comment( $comment, $depth, $args ) {
+		// Only call this local version for comments that are webmention based.
+		if ( 'webmention' !== get_comment_meta( $comment->comment_ID, 'protocol', true ) ) {
+			parent::comment( $comment, $depth, $args );
+			return;
+		}
+
+		$tag = ( 'div' === $args['style'] ) ? 'div' : 'li';
+
+		$cite  = apply_filters( 'webmention_cite', '<small>&nbsp;@&nbsp;<cite><a href="%1s">%2s</a></cite></small>' );
+		$url = get_url_from_webmention( $comment );
+		$host = wp_parse_url( $url, PHP_URL_HOST );
+		$host = preg_replace( '/^www\./', '', $host );
+		$type = get_webmention_comment_type_attr( $comment->comment_type, 'class' );
+		if ( 'comment' === $comment->comment_type ) {
+			$type = 'p-comment';
+		}
+
+		$commenter          = wp_get_current_commenter();
+		$show_pending_links = ! empty( $commenter['comment_author'] );
+
+		if ( $commenter['comment_author_email'] ) {
+			$moderation_note = __( 'Your comment is awaiting moderation.', 'default' );
+		} else {
+			$moderation_note = __( 'Your comment is awaiting moderation. This is a preview; your comment will be visible after it has been approved.', 'default' );
+		}
+		?>
+		<<?php echo $tag; ?> id="comment-<?php comment_ID(); ?>" <?php comment_class( $this->has_children ? 'parent' : '', $comment ); ?>>
+			<article id="div-comment-<?php comment_ID(); ?>" class="comment-body h-cite <?php echo $type; ?>">
+				<footer class="comment-meta">
+					<div class="comment-author vcard h-card u-author">
+						<?php
+						if ( 0 != $args['avatar_size'] ) {
+							echo get_avatar( $comment, $args['avatar_size'] );
+						}
+						?>
+						<?php
+						$comment_author = get_comment_author_link( $comment );
+
+						if ( '0' == $comment->comment_approved && ! $show_pending_links ) {
+							$comment_author = get_comment_author( $comment );
+						}
+
+						printf(
+							/* translators: %s: Comment author link. */
+							__( '%s <span class="says">says:</span>', 'default' ),
+							sprintf( '<b class="fn">%s</b>', $comment_author )
+						);
+						if ( ! empty( $cite ) && 'webmention' === get_comment_meta( $comment->comment_ID, 'protocol', true ) ) {
+							printf( $cite, $url, $host );
+						}
+
+						?>
+					</div><!-- .comment-author -->
+
+					<div class="comment-metadata">
+						<?php
+						// Allow arbitrary additions to comment metadata.
+						do_action( 'webmention_comment_metadata', $comment );
+						printf(
+							'<a class="u-url" href="%s"><time class="dt-published" datetime="%s">%s</time></a>',
+							esc_url( get_comment_link( $comment, $args ) ),
+							get_comment_time( DATE_W3C ),
+							sprintf(
+								/* translators: 1: Comment date, 2: Comment time. */
+								__( '%1$s at %2$s' ),
```

Comment:
```suggestion
								__( '%1$s at %2$s', 'webmention' ),
```

Reply:
This is actually from the original, so it should be ,'default'